types/pizzicato/index.d.ts

```diff
@@ -0,0 +1,566 @@
+// Type definitions for pizzicato 0.6
+// Project: https://alemangui.github.io/pizzicato/
+// Definitions by: Matthew Soulanille <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/pizzicato/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me